src/Agent/NewRelic/Profiler/Profiler/CorProfilerCallbackImpl.h

```diff
@@ -1125,7 +1126,6 @@ namespace NewRelic { namespace Profiler {
                     // Imbue with locale and codecvt facet is used to allow the log file to write non-ascii chars to the log
                     nrlog::StdLog.get_dest().imbue(std::locale(std::locale::classic(), new std::codecvt_utf8<wchar_t>));
                     nrlog::StdLog.get_dest().exceptions(std::wostream::failbit | std::wostream::badbit);
-                    LogInfo("Logger initialized.");
```

Comment:
Is this log message accurate? Should we keep this log message?

Reply:
It's accurate but probably not useful enough to warrant a whole cycle of me putting it back, updating the nuget package and re-running the workflows for this PR... ;-)